packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -1075,3 +1075,9 @@ String _getTargetPlatformByLocalEnginePath(String engineOutPath) {
   }
   return result;
 }
+
+extension on BuildInfo {
+  /// the flavor name in the output files is lowercased (see flutter.gradle),
+  /// so use the lower cased flavor nam to compute the output file name
+  String get lowerCasedFlavor => flavor?.toLowerCase();
```

Comment:
what about adding this to `BuildInfo` directly?

Reply:
done